python-avd/pyavd/_eos_designs/fabric_documentation_facts/__init__.py

```diff
@@ -260,9 +269,8 @@ def all_connected_endpoints(self) -> dict[str, list]:
                         "fabric_port": ethernet_interface["name"],
                         "description": get(ethernet_interface, "description", default="-"),
                         "shutdown": default(get(ethernet_interface, "shutdown"), get(port_channel_interface, "shutdown"), "-"),
-                        "type": default(get(port_channel_interface, "type"), get(ethernet_interface, "type"), "-"),
-                        "mode": default(get(ethernet_interface, "mode"), get(port_channel_interface, "mode"), "-"),
-                        "vlans": default(get(ethernet_interface, "vlans"), get(port_channel_interface, "vlans"), "-"),
+                        "mode": default(get(ethernet_interface, "switchport.mode"), get(port_channel_interface, "switchport.mode"), "-"),
+                        "vlans": default(self.get_interface_vlans(ethernet_interface), self.get_interface_vlans(port_channel_interface), "-"),
```

Comment:
This will execute the helper funciton on both ethernet_interface and port_channel_interface even if there are vlans found on the ethernet_interface.
It would be slightly more efficient to change it to `get_vlans(ethernet_interface, port_channel_interface)` and then only parse the port_channel_interface if no vlans were found.
Or even simpler, change the documentation to have a column for access-vlan / native-vlan and a column for allowed vlans.

Reply:
Added separate column for `access_vlan` and `allowed_vlans`